web/src/pages/Settings/Zones/Zones.jsx

```diff
@@ -108,19 +123,26 @@ const ZoneListItem = ({ zone }) => {
                     </div>
                 )}
             </div>
+            <StatusBar
+                open={alertOpen}
+                onClose={() => {setAlertOpen(false);}}
+                status={alertSuccess}
+                text={alertText}
+            />
         </ListItemButton>
     );
 };
 ZoneListItem.propTypes = {
     zone: PropTypes.any.isRequired,
 };
 
+
 const Zones = ({ onClose }) => {
     const zones = useStatusStore((state) => state.status.zones);
 
     const listItems = zones.map((zone) => {
         return(
-            <>
+            < >
```

Comment:
?

Reply:
I was having trouble along the way in this PR and my console was full of warnings and errors relating to how these empty tags don't have keys
I briefly turned them into diva and when that didn't help me I reverted the change, but clearly not entirely